test/unified_format.py

```diff
@@ -1229,6 +1235,16 @@ def _databaseOperation_createCommandCursor(self, target, **kwargs):
 
         return cursor
 
+    def kill_all_sessions(self):
+        if getattr(self, "client", None) is None:
```

Comment:
Why would client be None here? Should we be using the client_context.client?

Reply:
It was None on two of the tests that were testing the spec runner itself without creating a client.

Reply:
Oh I see it's TestCreateEntities.